### PR TITLE
Base the profile-and-properties itest on Java code rather than build output  Det

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
@jpkrohling could you please review and eventually merge?

c61cff2 is an important bugfix due to which I'd release after this PR is merged.

File: srcdeps-maven-quickstarts/srcdeps-mvn-git-profile-and-properties-quickstart/jar/src/main/java/org/l2x6/srcdeps/quickstarts/mvn/git/prop/profile/GreetingConsumer.java [hunk 1]
```diff
@@ -14,10 +14,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import org.codehaus.plexus.util.FileUtils;
+package org.l2x6.srcdeps.quickstarts.mvn.git.prop.profile;
+
+public class GreetingConsumer {
+
+    public String retrieveGreeting() {
+        return new org.l2x6.maven.srcdeps.itest.api.generated.GeneratedGreeter().getGreeting();
```
Comment (jpkrohling) [inline srcdeps-maven-quickstarts/srcdeps-mvn-git-profile-and-properties-quickstart/jar/src/main/java/org/l2x6/srcdeps/quickstarts/mvn/git/prop/profile/GreetingConsumer.java:22]: Import `GeneratedGreeter`?

**Review by jpkrohling — APPROVED** [review]
LGTM!

File: srcdeps-maven-quickstarts/srcdeps-mvn-git-profile-and-properties-quickstart/jar/src/main/java/org/l2x6/srcdeps/quickstarts/mvn/git/prop/profile/GreetingConsumer.java (hunk 1 above)
Comment (ppalaga) [reply srcdeps-maven-quickstarts/srcdeps-mvn-git-profile-and-properties-quickstart/jar/src/main/java/org/l2x6/srcdeps/quickstarts/mvn/git/prop/profile/GreetingConsumer.java:22]: No matter - there is only one occurrence. It came due to the fact that I did not have `GeneratedGreeter` in the class path of the project and I had to write it all by hand.

**Pull request merged**